<?xml version="1.0" encoding="utf-8"?>

<launch>

  <arg name="rgb_image" default="/d400/color/image_raw" />
  <arg name="depth_image" default="/d400/aligned_depth_to_color/image_raw" />
  <arg name="camera_info" default="/d400/depth/camera_info" />

  <arg name="name_bag_file" default="$(find pipe_inspection_drone)/scripts/tracking" />

  <node name="tracking_record" pkg="pipe_inspection_drone" type="tracking_record.sh"  args="$(arg name_bag_file) $(arg rgb_image) $(arg depth_image) $(arg camera_info) /pipe_inspection_drone/image_binary_mask/compressed /pipe_inspection_drone/image_rgb_axis/compressed /pipe_inspection_drone/pose /mavros/local_position/pose /mavros/setpoint_raw/local" output="screen">  
  </node>

</launch>
